measures/ResStockArguments/measure.rb

```diff
@@ -640,6 +640,7 @@ def run(model, runner, user_arguments)
 
       # Apply adjustment to infiltration value
       args['air_leakage_value'] *= exposed_wall_area_ratio
+      args['air_leakage_multifamily_value_type'] = HPXML::InfiltrationTestGuarded
```

Comment:
Why not specify in options_lookup.tsv?

Reply:
Certainly could. Seems more appropriate alongside all the infiltration adjustment stuff, no?

Reply:
No, the adjustments are not converting from compartmentalization values to guarded values. They are just increasing some units slightly and decreasing others, while keeping the overall infiltration for the MF building unchanged.

Reply:
But we still want to set to "guarded test", right? Seems like putting that here implies were not unintentionally missing setting it to "compartmentalization test".